Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -544,7 +544,7 @@ noStdin                  | Do not use `--read-args-from-stdin` for the wkhtmltop
 Key                      | Description
 -------------------------|-----------------------------
 filePath                 | The path and file name of a wkhtmltopdf.exe compatible executable.
-additionalArguments      | Additional arguments that should be passed to the wkhtmltopdf executable.
+additionalArguments      | Additional arguments that should be passed to the wkhtmltopdf executable. For example, pass `--enable-local-file-access` if you are building on a local file system. This will ensure that the supporting *.js and *.css files are loaded when rendering the HTML being converted to PDF.
```

Comment:
Should we mention `version >= 0.12.6`, or do you think it is a general suggestion regardless of wkhtmltopdf version?

Reply:
I think just as a general suggestion would be good, especially since this emulates the behavior prior to 0.12.6.

Reply:
So, no harm in passing this value if someone is using an older wkhtmlpdf.